### DISCUSSION TITLE
Only get a list of your own defined keymaps and commands

### DISCUSSION THREAD

**Discussion by Lilja** [mrjones2014/legendary.nvim#470] [Q&A] [answered]
I really like legendary, but sometimes I just really want to search among my own key binds, as I tend to write a niche telescope extension.

If type `:Legendary` and search for a keyword that I know of is contained in the description property I set up, it shows up with other "native"(?) key binds such as `gk`, `gj`, etc.

See this screenshot:
![image](https://github.com/user-attachments/assets/e6eab5cb-35f7-493f-8b84-4c1a636dd96e)

Is there a way only to show the commands defined in:

```lua
require("legendary").setup({
  keymaps = {
  -- Bufjump, https://github.com/kwkarlwang/bufjump.nvim
  {
    "<leader>o",
    function()
        bufjump.backward()
    end,
    description = "Buffer back",
   },
})
```

**Comment by mrjones2014** [comment, marked as answer]
It’s in the README. Set `include_builtin = false` in config

**Reply by Lilja** [reply]
Thanks Mat! I think in my quest to find the proper option I searched for "custom", as in "only show my own custom commands", and I never found it.
Now it makes total sense.
